.changeset/nasty-kiwis-talk.md

```diff
@@ -0,0 +1,11 @@
+---
+"wrangler": minor
+---
+
+Include runtime types in the output of `wrangler types` by default
+
+`wrangler types` will now produce one file that contains both `Env` types and runtime types based on your compatibility date and flags. This is located at `worker-configuration.d.ts` by default.
```

Comment:
Worth taking this opportunity to rename this file, maybe to `cloudflare.d.ts`? cc @irvinebroque if you have any thoughts here